src/components/BlogList.vue

```diff
@@ -65,24 +31,23 @@ onMounted(() => {
   fetchArticles();
 });
 
-watch(
-  () => props.searchQuery,
-  (newQuery) => {
-    page.value = 1;
-    fetchArticles();
-  }
-);
+function formatDate(timestamp: number) {
+  const date = new Date(timestamp * 1000);
+  return new Intl.DateTimeFormat('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  }).format(date);
+}
```

Comment:
Let's move this file a `utility.js` file. Export the function from that file and use it here. Thanks

Reply:
Hello bhaia, I've moved the function to a different file named utility.ts